galaxy/values.yaml

```diff
@@ -49,6 +49,16 @@ persistence:
   accessMode: ReadWriteMany
   size: 10Gi
   mountPath: /galaxy/server/database
+  customVolume: {}
```

Comment:
What about about using this mechanism instead?
https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/preexisting-pd

Reply:
https://github.com/CloudVE/projman-helm/blob/master/projman/values.yaml#L30 yeh I was thinking about just adding the same mechanism I added in projman (https://github.com/CloudVE/projman-helm/blob/master/projman/values.yaml#L30) to the galaxy chart itself. It should work for both ceph and rclone. I haven't had a chance to test but I can try it out next week, should be straightforward.